app/controllers/quotes_controller.rb

```diff
@@ -3,6 +3,7 @@ class QuotesController < ApplicationController
 
   def show
     @quote = Quote.find params[:id]
+    @vote = @quote.votes.find_by(user: current_user)
```

Comment:
It would break the quote show page without if I'm remembering right.  Maybe I added more than I should have by including view functionality at the same time?  Was trying to imitate the quote controller commit I'd made before.